# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../trac-doxygen/hotfix.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./doxygentrac/doxygentrac.py.orig	2006-12-13 11:16:55.000000000 -0500
+++ ./doxygentrac/doxygentrac.py	2006-12-13 11:17:14.000000000 -0500
@@ -87,7 +87,7 @@
         if req.perm.has_permission('DOXYGEN_VIEW'):
             # Return mainnav buttons.
             yield 'mainnav', 'doxygen', html.a(self.title,
-                                               href=req.href.doxygen())
+                                               href=req.href.doxygen() + '/' )
 
     # IRequestHandler methods
 
